### DISCUSSION TITLE
使用v2ray但是google还能看到源IP，其他网站正常。是缺少什么配置吗？

### DISCUSSION THREAD

**Discussion by etherun** [v2fly/v2ray-core#3029] [General]
![image](https://github.com/v2fly/v2ray-core/assets/141379558/1fb59b8c-95da-4fed-a43b-2a14f571b9b9)
查询IP已是国外的了
![image](https://github.com/v2fly/v2ray-core/assets/141379558/12826b4d-e1fd-4536-8126-6c1a3e7249f4)

**Comment by qingfengshi** [comment]
检查cookie，浏览器设置

**Reply by etherun** [reply]
老哥，cookie全部清掉了，换了firefox问题依旧啊
浏览器的Location也关掉了，还有哪些设置呢？ 感谢

**Comment by qingfengshi** [comment]
我也不知道了，之前遇到过，后面又自己好了
